src/Repository.js

```diff
@@ -55,6 +55,9 @@ export default class Repository {
   }
 
   get packageConfigs() {
+    if (this.lernaJson.useWorkspaces) {
+      return this.packageJson.workspaces;
```

Comment:
Yarn does not have a default (PR welcome ;), so it would boom there if this setting is missing.
There is a comment about `package.json/workspaces` being used instead of `packages` https://github.com/lerna/lerna#--use-workspaces